test/evergreen.yml

```diff
@@ -5689,8 +5689,6 @@ buildvariants:
       - ubuntu2004-medium
     - name: format-abort-recovery-stress-test
     - name: format-predictable-test
-    # FIXME-WT-10822
-    # - name: format-tiered-test
```

Comment:
@ajmorton This function is now redundant. Should we remove it in this pull request?

Reply:
Yep, good point 👍 
I've removed it and the resulting dead code in [8724093](https://github.com/wiredtiger/wiredtiger/pull/11819/commits/87240938727d8e5e29df7d55cfb31a121a52f39e)